src/go/wsl-helper/pkg/dockerproxy/mungers/containers_create_linux.go

```diff
@@ -310,18 +310,40 @@ func (b *bindManager) mungeContainersStartRequest(req *http.Request, contextValu
 
 	// Do bind mounts
 	for bindKey, target := range mapping {
-		mountDir := path.Join(b.mountRoot, bindKey)
+		mountPath := path.Join(b.mountRoot, bindKey)
 		logEntry := logrus.WithFields(logrus.Fields{
 			"container": templates["id"],
-			"bind":      mountDir,
+			"bind":      mountPath,
 			"target":    target,
 		})
-		err := os.MkdirAll(mountDir, 0o700)
-		if err != nil {
-			logEntry.WithError(err).Error("could not create mount directory")
-			return fmt.Errorf("could not create volume mount %s: %w", target, err)
+		hostPathStat, err := os.Stat(target)
```

Comment:
Would there be a TOCTTOU concern here? I guess not because it's all running in the same context as the user (because no matter the Linux-side user account, this is all running under the same Windows user account)?

Reply:
I don't think so.
1. If you run `docker run ... -v ...` concurrently from two different windows terminals, you'd be working in two separate containers anyway, right?
2. And if the path argument to `os.MkdirAll` already exists, it returns a `nil` error.